test/spec/desktop/windowHandleSize.js

```diff
@@ -2,7 +2,19 @@ import conf from '../../conf/index'
 
 describe('windowHandleSize', () => {
     it('should change window size of current window if no handle is given', async function () {
-        await this.client.windowHandleSize({ width: 500, height: 500 })
```

Comment:
It does not pass on master branch on my machine. And it cannot pass, since I've added a check that the URL ends with "/window/current/size", and in the current master branch it actually sends requests to "/window/[object Object]/size". You can see this in the Selenium server log, for example.